File: pset6/mario/mario.py
from cs50 import get_int

h = 0

# Continues to prompt the user for a height between 1 and 8 inclusive
while (h < 1 or h > 8):
    h = get_int("Height: ")

# Prints a pyramid of given height
for i in range(int(h)):
    h -= 1
    i += 1
    print(" " * h, end="")
    print("#" * i, end="")
    print("  ", end="")
    print("#" * i, end="")
    print("")

File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //assigns h to the height inputted by the user
    int h;

    //continues to prompt user until a positive number between 1 and 8 is entered
    do
    { 
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    //First loop dictates the number of rows and intiates them one by one
    for (int i = h; i > 0; i--)
    {
        //Secondary loops build each row, space by space and block by block
        for (int j = (i - 1); j > 0; j--)
        {
            printf(" ");
        }
        for (int k = (h - (i - 1)); k > 0; k--)
        {
            printf("#"); 
        } 
        printf("  ");
        for (int k = (h - (i - 1)); k > 0; k--)
        {
            printf("#"); 
        }
        printf("\n");
    }
}